src/renderer/features/wallets/ForgetWallet/model/__tests__/forget-wallet-model.test.ts

```diff
@@ -69,4 +112,43 @@ describe('features/ForgetModel', () => {
     expect(spyDeleteWallet).toBeCalledWith(2);
     expect(spyDeleteAccounts).toBeCalledWith([1, 2]);
   });
+
+  test('should delete proxied accounts, wallets and proxyGroups', async () => {
+    storageService.proxies.deleteAll = (ids: number[]) => new Promise((resolve) => resolve(ids));
```

Comment:
You need to use `jest.spyOn` for proper mocks, we got plenty of that across the project

`jest.spyOn` comes along with:
```javascript
 beforeEach(() => {
    jest.restoreAllMocks();
  });
```